tests/integration-tests/tests/performance_tests/test_startup_time.py

```diff
@@ -0,0 +1,125 @@
+
+    for metric in cluster_metrics["Metrics"]:
+        result = cw_client.get_metric_statistics(
+            Namespace="ParallelCluster",
+            MetricName="StartupTime",
+            Dimensions=metric["Dimensions"],
+            StartTime=datetime.now() - relativedelta(years=1),
+            EndTime=datetime.now(),
+            Period=30000,
+            Statistics=["Average"],
+            Unit="None",
+        )
+
+        logging.info(f"Results: {result}")
+
+        if result["Datapoints"]:
+            value = result["Datapoints"][0].get("Average")
+            data.append(value)
+    return data
+
+
+def get_metric(dimensions, cw_client):
+    value = None
+
+    result = cw_client.get_metric_statistics(
+        Namespace="ParallelCluster",
+        MetricName="StartupTime",
+        Dimensions=dimensions,
+        StartTime=datetime.now() - relativedelta(years=1),
+        EndTime=datetime.now(),
+        Period=30000,
+        Statistics=["Average"],
+        Unit="None",
+    )
+
+    logging.info(f"Results: {result}")
+
+    if result["Datapoints"]:
+        value = result["Datapoints"][0].get("Average")
+    return value
+
+
+def test_startup_time(pcluster_config_reader, clusters_factory, test_datadir, region, instance, os, scheduler):
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    logging.info("Cluster Created")
+
+    cw_client = boto3.client("cloudwatch", region_name=region)
+
+    instances = describe_cluster_instances(
+        cluster.name,
+        region,
+        filter_by_node_type="Compute",
+    )
+
+    performance_degradation = {}
+
+    for instance in instances:
+        instance_type = instance["InstanceType"]
+        instance_id = instance["InstanceId"]
+        logging.info(f"Type: {instance_type}")
+
+        # get new startup time
+
+        dimensions = [
+            {"Name": "OS", "Value": os},
+            {"Name": "InstanceID", "Value": instance_id},
+            {"Name": "ClusterName", "Value": cluster.name},
+            {"Name": "InstanceType", "Value": instance_type},
+        ]
+
+        value = get_metric(dimensions, cw_client)
+        logging.info(f"Value: {value}")
+
+        # get historical data
+        dimensions = [{"Name": "InstanceType", "Value": instance_type}, {"Name": "OS", "Value": os}]
+        data = get_data(dimensions, cw_client)
+        if value in data:
+            data.remove(value)
+
+        logging.info(f"Data of {instance_type}: {data}")
+
+        # evaluate data
+        if len(data) > MINIMUM_DATASET_SIZE and value:
+            degradation = evaluate_data(value, data)
+            if degradation:
+                performance_degradation[instance_type] = value
+
+    if performance_degradation:
+        degraded_instances = performance_degradation.keys()
+        pytest.fail(
+            f"Performance test results show performance degradation for the following instances: {degraded_instances}"
```

Comment:
I suggest to specify the magnitude of the degradation with the failure log.
If there is degradation, we would like to know how much is the degradation to make more informative decision. 

Example: If the degradation is really close to the level of tolerance we may end up saying that it's ok because the sample size is not enough to mitigate the variance.